app/controllers/sessions_controller.rb

```diff
@@ -3,11 +3,17 @@ def new
   end
   
   def create
-    user = User.find_by email: login_params[:email]
+    user = User.find_by(email: login_params[:email].downcase)
     
     if user&.authenticate(login_params[:password])
-      cookies.encrypted[:user_id] = user.id
-      redirect_to root_url, notice: t('.session_created')
+      if user.activated?
+        #log_in user
+        cookies.encrypted[:user_id] = user.id
```

Comment:
If the `log_in(user)` method is available via the `Authentication` concern, this code would be better if you use that (like in the commented-out part of code).